src/backoffice/shared/components/workspace-property/workspace-property.element.ts

```diff
@@ -213,9 +232,11 @@ export class UmbWorkspacePropertyElement extends UmbLitElement {
 		return html`
 			<umb-workspace-property-layout
 				id="layout"
+				alias="${ifDefined(this._alias)}"
 				label="${ifDefined(this._label)}"
 				description="${ifDefined(this._description)}">
 				${this._renderPropertyActionMenu()}
+				<p slot="description">${this._variantDisplayName}</p>
```

Comment:
I get a warning here saying that slot=description does not exist. Perhaps missing @slot jsdoc?

Reply:
That right, I have correct that now